src/main/java/hw3/composite/LogField.java

```diff
@@ -0,0 +1,41 @@
+package hw3.composite;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+
+public class LogField extends AbstractPageComposite {
+
+    @FindBy(xpath = "//li[contains(text(),'Water')]")
+    private WebElement waterLogs;
+    @FindBy(xpath = "//li[contains(text(),'Wind')]")
+    private WebElement windLogs;
+    @FindBy(xpath = "//li[contains(text(),'Selen')]")
+    private WebElement selenLogs;
+    @FindBy(xpath = "//li[contains(text(),'Yellow')]")
+    private WebElement yellowLogs;
```

Comment:
You could replace these elements to the `List<WebElement>`

Reply:
Created List<WebElement> for all log elements